pack_and_terraform_aws.bash

```diff
@@ -147,21 +147,33 @@ if echo "$RUNTF" | grep -iq "^y"; then
     if [ -z "$AWS_S3_ACCESS_KEY" ]; then
         read -p "Enter your access key to S3 bucket for saving terraform state: " AWS_S3_ACCESS_KEY
     fi
+    # Terraform variable: 
+    #  https://www.terraform.io/docs/configuration/environment-variables.html#tf_var_name
+    export TF_VAR_access_key="${AWS_S3_ACCESS_KEY}"
```

Comment:
let's not do this? should be fixed [here](https://github.com/uc-cdis/cloud-automation/pull/93/commits/1943eb611b69e5dd168c3e3f7b632b33fe07e3a1#diff-40a95469d103a81a11be31908b3fc11eR264) to use those creds for `init` but exporting them as tf vars are error prone because AWS_ACCESS_KEY might be different from AWS_S3_ACCESS_KEY, and we might not use the creds to access s3 for terraform apply

Reply:
The reason we were setting those variales was to plug into the terraform.tfvars file that I erased which has lines like:
      access_key = "${AWS_ACCESS_KEY}"
  Terraform already has a mechanism to set variable via environment variables named TF_VAR_[name]
...

Reply:
yeah I know, you are exporting the **s3** access_key to the TF_VAR_access_key which can be used by both the s3 backend and deployment. but we have both AWS_ACCESS_KEY and AWS_S3_ACCESS_KEY so I I think it's cleaner to use the var files and specify which var file to use instead of tf variables